topics/performance.md

```diff
@@ -1,12 +0,0 @@
-# Performance
-
-During development, the focus should be on readability, not on locally optimized performance.
-It's easier to optimize readable code than to make code with good performance readable.
-If the code is unreadable and not performant, what do you do?
-
-* Nevertheless, user actions should complete within 200 ms. If they're slower, try to optimize your code.
```

Comment:
The 200ms is important to external developers and is still relevant. Can we add it to some other section?

Reply:
I restored the whole file again.